Makefile

```diff
@@ -28,9 +28,9 @@ TAG ?= devel
 TOOLCHAIN_REPOSITORY ?= cartesi/toolchain
 TOOLCHAIN_TAG ?= 0.15.0
 KERNEL_VERSION ?= 5.15.63-ctsi-2
-KERNEL_SRCPATH := linux-$(KERNEL_VERSION).tar.gz
-RISCV_PK_VERSION ?= 1.0.0-ctsi-1
-RISCV_PK_SRCPATH := riscv-pk-$(RISCV_PK_VERSION).tar.gz
+KERNEL_SRCPATH := dep/linux-${KERNEL_VERSION}.tar.gz
+OPENSBI_VERSION ?= opensbi-1.2-ctsi-y
```

Comment:
We need to upgrade to OpenSBI 1.3.1, because I hit the following issue that was fixed in newer OpenSBI:
- https://github.com/cartesi/opensbi/issues/2

Also QEMU 8.1.1 is already using OpenSBI 1.3.1 for emulating RISC-V